.golangci.yml

```diff
@@ -8,6 +8,13 @@ run:
   # activate after switching to go modules (#131, #190)
   # modules-download-mode: vendor
 
+  # TODO: This is hard-coded here because of https://github.com/golangci/golangci-lint/issues/2649#issuecomment-1070528726
+  # golangci-lint had issues with go1.18 so some of the linters are disabled. As we are not using go1.18 features
```

Comment:
I have successfully made it work for 1.18 in NT and it actually works in most cases.
Are there some specific blocks why we can't use 1.18? A lot of fixes were done since the post of mentioned comment. Also, we do not need most if not all of the indicated linters.

Reply:
I'd suggest rather disabling those linters then moving on with version discrepancy. It is a good question if we need those enabled at all. If not, please make a follow-up issue to fix that later.